Makefile.am

```diff
@@ -312,7 +312,8 @@ $(BACKEND_TARGET): $(top_srcdir)/$(SCRIPTSDIR)/setup_compiler.sh | $(RESULTSDIR)
 	    --includedir "$(WDLC_INCLUDEDIR)" \
 	    --libdir "$(WDLC_LIBDIR)" \
 	    --srcdir "$(top_srcdir)" \
-	    --output "$(RESULTSDIR)"
+	    --output "$(RESULTSDIR)" \
+	    --local-packages "$(LOCAL_SITE_PACKAGES_DIR)"
```

Comment:
Is "LOCAL_SITE_PACKAGES_DIR" a Python-standard environment variable?

Reply:
No, it's something I added. I couldn't find anything standard to model after in my Google searches.